src/tests/multihost/basic/test_ldap.py

```diff
@@ -11,6 +11,7 @@
 from sssd.testlib.common.utils import SSHClient
 import pytest
 import textwrap
+import paramiko
```

Comment:
Why is this needed?

Reply:
The same reason as before, this file is using `paramiko` and thus it should import it.